README.md

```diff
@@ -75,6 +75,20 @@ On MacOS, you should replace the default `llvm` with the one from `brew`:
 brew install llvm
 ```
 
+> [!TIP]
```

Comment:
This is very setup specific and I don't see the correlation between the repo and this tip. Not the repo's responsibility to explain how one should install Rust IMO. For example, I'm on MacOS and didn't install Rust using `brew`.

Reply:
I do agree it's kinda setup-specific, but I had to go through the setup myself and do some digging 🙈 and since there was already a "MacOS specific" section, I thought it would be a nice addition.

Maybe we could have those instructions somewhere else in this case? To avoid polluting  the main README?

Reply:
As discussed together, I've added a `DEVELOPMENT.md` separate file.